exercises/src/test/scala/fpinscala/LazinessSpec.scala

```diff
@@ -0,0 +1,153 @@
+package fpinscala
+
+import fpinscala.laziness.{ Stream, Empty, Cons}
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class LazinessSpec extends AnyFlatSpec with Matchers {
+
+
+  it should "convert a stream to a list" in {
+    Stream(1, 2, 3, 4).toList() shouldBe List(1, 2, 3, 4)
+    Stream().toList() shouldBe Nil
+  }
+
+  it should "take" in {
+    Stream(1, 2, 3, 4).take(2).toList() shouldBe List(1, 2)
+    Stream(1, 2, 3, 4).take(1).toList() shouldBe List(1)
+    Stream(1, 2, 3, 4).take(0).toList() shouldBe Nil
+  }
+
+  it should "drop" in {
+    Stream(1, 2, 3, 4).drop(2).toList() shouldBe List(3, 4)
+    Stream(1, 2, 3, 4).drop(1).toList() shouldBe List(2, 3, 4)
+    Stream(1, 2, 3, 4).drop(0).toList() shouldBe List(1, 2, 3, 4)
+  }
```

Comment:
The code looks like it should work if `n` is greater than the length of the stream (unlike `take`), but probably good to add a test for it still.